src/modules/regions/regions.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
Please stick to the agreed field name for status codes which is status_code

Reply:
i've fixed that